src/views/mixed-content/top-countries-with-more-websites-with-mixed-content/elements.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+const ChartContainer = styled.div`
+  height: 300px;
+  display: flex;
+  justify-content: center;
+  align-items: center
+`;
+
```

Comment:
Question: could we have only one element.js file at /views and use it in all charts? I think it's useful for all visualizations and I don't see any differences between this one and the one at /mixed-content-percentage-histogram

Reply:
Maybe a good idea, but as per #9, most of the ChartContainers might change.

At this point, the code is the same, and could feel repeated,  but their scope is different. In my opinion, we could wait 'till we reorganize the view to see if there's a pettern